lib/ruby_lsp/internal.rb

```diff
@@ -19,14 +19,6 @@
 require "prism/visitor"
 require "language_server-protocol"
 
-# Prism v0.26.0 introduced generics for ParseResult, but it causes some problems so the intention is to remove it.
-# Once that is done, we can remove this patch.
-module Prism
-  class ParseResult
-    extend T::Generic
-  end
-end
```

Comment:
Actually we will need to conditionally include this... will think how to handle that.